assets/js/state/sagas/operations.js

```diff
@@ -82,7 +86,27 @@ export function* completeOperation({ payload }) {
   }
 }
 
+export function* updateRunningOperation({ payload }) {
+  const { groupID } = payload;
+
+  try {
+    const {
+      data: { items: operations, total_count: totalCount },
+    } = yield call(getOperationExecutions, {
+      group_id: groupID,
+      items_per_page: 1,
+    });
+    if (totalCount === 1 && operationRunning(operations[0])) {
+      const operation = getOperationInternalName(operations[0].operation);
```

Comment:
If we could destructure that in the signature the first item could be better, wdyt?

Reply:
Done